src/main/webapp/js/datahandlers/collegepagedatahandler.js

```diff
@@ -19,114 +19,52 @@
 
 goog.module('datahandlers.collegepage');
 
-const {collegepage} = goog.require('finscholar.collegepageview.templates');
+const {CollegeQueryBuilder, ID, NAME, ACCEPTANCE_RATE, ACT_SCORE, ANNUAL_COST, 
+    FIRST_NET_COST, SECOND_NET_COST, THIRD_NET_COST, FOURTH_NET_COST, 
+    FIFTH_NET_COST, MEDIAN_DEBT} = goog.require('datahandlers.collegequerybuilder');
 
-/** This constant is the endpoint to send a fetch request to. */
-const COLLEGE_SERVLET_ENDPOINT = '/college-data';
-const ERROR_MESSAGE_JSON_UNDEFINED = 'JSON object undefined.';
-const OCCURRENCE_JSON_UNDEFINED =
-    'A network error has occurred. Failed to load college data.';
-const ACTION_JSON_UNDEFINED =
-    'Please reload the page or select a different college.';
-const OCCURRENCE_SERVER_ERROR =
-    'A database error occurred. Failed to render college data.';
-const ACTION_SERVER_ERROR = 'The database failed to retrieve the college. \
-      This college may not exist. Please reload the page or select a different college.';
+class CollegeDataHandler {
```

Comment:
You probably want to wait for Lorraine's refactor and base data handler class to be merged before moving forward with this.